gwcli/storage.py

```diff
@@ -478,48 +573,40 @@ def get_meta_data_tcmu(self):
             with cluster.open_ioctx(self.pool) as ioctx:
                 with rbd.Image(ioctx, self.image) as rbd_image:
                     self.size = rbd_image.size()
-                    self.size_h = human_size(self.size)
                     self.features = rbd_image.features()
-                    self.feature_list = self._get_features()
-
-        # update the parent's disk info map
-        disk_map = self.parent.disk_info
-
-        disk_map[self.image_id]['size'] = self.size
-        disk_map[self.image_id]['size_h'] = self.size_h
-
-    def get_meta_data_krbd(self):
-        """
-        KRBD based method to get size and rbd features information
-        :return:
-        """
-        # image_path is a symlink to the actual /dev/rbdX file
-        image_path = "/dev/rbd/{}/{}".format(self.pool, self.rbd_image)
-        dev_id = os.path.realpath(image_path)[8:]
-        rbd_path = "/sys/devices/rbd/{}".format(dev_id)
-
-        try:
-            self.features = readcontents(os.path.join(rbd_path, 'features'))
-            self.size = int(readcontents(os.path.join(rbd_path, 'size')))
-        except IOError:
-            # if we get an ioError here, it means the config object passed
-            # back from the API is out of step with the physical configuration
-            # this can happen after a purge_gateways ansible playbook run if
-            # the gateways do not have there rbd-target-gw daemons reloaded
-            error_msg = "The API has returned disks that are not on this " \
-                        "server...reload rbd-target-api?"
-
-            self.logger.critical(error_msg)
-            raise GatewayError(error_msg)
-        else:
-
-            self.size_h = human_size(self.size)
-
-            # update the parent's disk info map
-            disk_map = self.parent.disk_info
 
-            disk_map[self.image_id]['size'] = self.size
-            disk_map[self.image_id]['size_h'] = self.size_h
+    # def get_meta_data_krbd(self):
```

Comment:
Nit: feel free to drop this code -- it's saved for posterity in the git history.